src/Router/Route.php

```diff
@@ -184,6 +196,41 @@ public function middlewares(array $middlewares = []): Route
         return $this;
     }
 
+	public function cacheable(bool $shouldCache, int $ttl = null): Route
+	{
+		if (!$ttl) {
+			$ttl = config()->get('view_cache.ttl', 300);
+		}
+
+		if (!empty(session()->getId())){
```

Comment:
if `session()->getId()` is empty the the whole function just returns `$this`, so the above code is unnecessary

```
if (!$ttl) {
			$ttl = config()->get('view_cache.ttl', 300);
		}
```

Reply:
Use early return instead of nesting